### PR TITLE
Created example of how to make a simple inbound TTS

### PR DESCRIPTION
**Pull request by tommorris**
This is to be used as a building block in the new documentation.

### CODE REVIEW

File: voice/inbound_tts_call.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from flask import Flask, request, jsonify
+from pprint import pprint
```
Comment (judy2k) [inline voice/inbound_tts_call.py]: `pprint` imported but not used?

File: voice/inbound_tts_call.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    ncco = [{
+        "action": "talk",
+        "text": "Hello World",
+    }]
+    return jsonify(ncco)
+
+
+@app.route('/callback', methods=['POST'])
+def callback_handle():
+    print request
```
Comment (judy2k) [inline voice/inbound_tts_call.py]: I'd prefer it if we import print_function from __future__ and use parentheses here.

**Comment by leggetter** [comment]
@tommorris @judy2k - can we have this resolved please?

**Comment by sammachin** [comment]
closing as this is redundant now

**Pull request closed without merging**